CHANGELOG.md

```diff
@@ -1,5 +1,82 @@
+* compute: removed the `enable_display` field from `google_compute_instance_template` ([#10410](https://github.com/hashicorp/terraform-provider-google/pull/10410))
+* compute: removed the `update_policy.min_ready_sec` field from `google_compute_instance_group_manager`, `google_compute_region_instance_group_manager` ([#10410](https://github.com/hashicorp/terraform-provider-google/pull/10410))
+* container: `instance_group_urls` has been removed in favor of `node_pool.managed_instance_group_urls` ([#10442](https://github.com/hashicorp/terraform-provider-google/pull/10442))
+* container: changed default for `enable_shielded_nodes` to true for `google_container_cluster` ([#10403](https://github.com/hashicorp/terraform-provider-google/pull/10403))
+* container: changed `master_auth.client_certificate_config` to required ([#10441](https://github.com/hashicorp/terraform-provider-google/pull/10441))
+* container: removed `master_auth.username` and `master_auth.password` from `google_container_cluster` ([#10441](https://github.com/hashicorp/terraform-provider-google/pull/10441))
+* container: removed `workload_metadata_configuration.node_metadata` in favor of `workload_metadata_configuration.mode` in `google_container_cluster` ([#10400](https://github.com/hashicorp/terraform-provider-google/pull/10400))
+* container: removed the `pod_security_policy_config` field from `google_container_cluster` ([#10410](https://github.com/hashicorp/terraform-provider-google/pull/10410))
+* container: removed the `workload_identity_config.0.identity_namespace` field from `google_container_cluster`, use `workload_identity_config.0.workload_pool` instead ([#10410](https://github.com/hashicorp/terraform-provider-google/pull/10410))
+* project: removed ability to specify `bigquery-json.googleapis.com`, the provider will no longer convert it as the upstream API migration is finished. Use `bigquery.googleapis.com` instead. ([#10370](https://github.com/hashicorp/terraform-provider-google/pull/10370))
+* provider: changed `credentials`, `access_token` precedence so that `credentials` values in configuration take precedence over `access_token` values assigned through environment variables ([#10393](https://github.com/hashicorp/terraform-provider-google/pull/10393))
+* provider: removed redundant default scopes. The provider's default scopes when authenticating with credentials are now exclusively "https://www.googleapis.com/auth/cloud-platform" and "https://www.googleapis.com/auth/userinfo.email". ([#10374](https://github.com/hashicorp/terraform-provider-google/pull/10374))
+* pubsub: removed `path` field from `google_pubsub_subscription` ([#10424](https://github.com/hashicorp/terraform-provider-google/pull/10424))
+* resourcemanager: `org_id`, `folder_id` are unset when removed from config in `google_project` ([#10373](https://github.com/hashicorp/terraform-provider-google/pull/10373))
```

Comment:
do we need to start with a verb here?

Reply:
or is this redundant with the entry just below it?

Reply:
I could't think of a good phrasing when writing this! I kept it separate from the one below intentionally, since there are two changes from an end user's perspective imo. Maybe this?

> resourcemanager: made `google_project` remove `org_id` and `folder_id` from state when they are removed from config

Reply:
Ok, gotcha! I actually like that phrasing a bit better, I was unsure if their being unset was what was fixed or if it _was_ the fix.